lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -13,6 +13,10 @@ function canMultiplyLikeTermConstantNodes(node) {
     return false;
   }
 
+  if (args.every(arg => ConstantOrPowerTerm.getExponentNode(arg).value === 1)) {
```

Comment:
actually instead of this I think it'd be better to check that all of them don't have exponents

 i.e. `args.every(arg => !Node.Type.isOperator(arg, '^'))` since they'd all be constants and not power terms. I think it's a bit easier to understand what's going on

what do you think?